doc/api/crypto.md

```diff
@@ -3510,6 +3510,67 @@ Computes the Diffie-Hellman secret based on a `privateKey` and a `publicKey`.
 Both keys must have the same `asymmetricKeyType`, which must be one of `'dh'`
 (for Diffie-Hellman), `'ec'` (for ECDH), `'x448'`, or `'x25519'` (for ECDH-ES).
 
+### `crypto.hash(algorith, data[, outputEncoding])`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+* `algorithm` {string|undefined}
+* `data` {string|ArrayBuffer|Buffer|TypedArray|DataView} When `data` is a
```

Comment:
Set the `ArrayBuffer` type data will throw TypeError.

```bash
TypeError: The "input" argument must be an instance of Buffer, TypedArray, or DataView. Received an instance of ArrayBuffer
      at hash (node:internal/crypto/hash:200:5)
```

Reply:
@fengmk2 can you open a separate issue please?

Reply:
sure

Reply:
https://github.com/nodejs/node/pull/52069